pages/blog/index.vue

```diff
@@ -102,4 +106,12 @@ export default {
 .post-card-readtime {
   @apply text-sm
 }
+@media only screen and (max-width: 320px) {
```

Comment:
We can introduce new media query in tailwind.config.js and then use `@screen` https://tailwindcss.com/docs/breakpoints/#app

Reply:
like so ?
```js
// tailwind.config.js
module.exports = {
  theme: {
    screens: {
      'xs': '320px'
    }
  }
}
```

Reply:
Yes :+1: or this also should work, since we already using `extend`
```
// tailwind.config.js
module.exports = {
  theme: {
    extend: {
       screens: {
         'xs': '320px'
       }
    }
  }
}
```

Reply:
I noticed this issue https://github.com/tailwindcss/tailwindcss/issues/1319#issuecomment-576016505 so I didn't use `extend`

Reply:
Thanks, didn't know about that!